docs/reference/search/request/highlighting.asciidoc

```diff
@@ -261,7 +261,7 @@ type:: The highlighter to use: `unified`, `plain`, or `fvh`. Defaults to
 * <<set-highlighter-type, Set highlighter type>>
 * <<configure-tags, Configure highlighting tags>>
 * <<highlight-source, Highlight source>>
-* <<matched-fields, Combine matches on multiple fields>>
+* <<matched-fields, Combine matches on multiple fields using matched_fields>>
```

Comment:
Hi @Austio, thanks for the suggestion. I understand that his might increase the likelihood of people searching for examples of how to use `matched_field` actually find it, but I would rather keep the list of links short and descriptive rather than already mixing in the "how" and implementation details like the parameter name used for a certain thing. Instead, maybe it would be possible to add a link to the example section when the `matched_fields` parameter is described earlier in L196? Would that have worked for you?

Reply:
@cbuescher hey that sounds good.  Will update that today.  Sorry missed this in my messages last week.